source/request-response/subscription_manager.c

```diff
@@ -344,6 +383,10 @@ static int s_rr_activate_idle_subscription(
     struct aws_rr_subscription_record *record) {
     int result = AWS_OP_SUCCESS;
 
+    if (record->poisoned) {
+        return AWS_OP_SUCCESS;
```

Comment:
Should we log that this is a poisoned success? It may help track down poison related issues in the future.

Reply:
Yeah that's a good idea.